index.html

```diff
@@ -594,52 +596,57 @@
+Given the required multiple-scattering albedo $\mathbf{E}_\mathrm{multi-scatter}$ according to the formula above, then in principle, the single-scattering albedo $\boldsymbol{\alpha}$ which generates the required $\mathbf{E}_\mathrm{multi-scatter}$ can be determined. We do not require any particular theoretical formula for $\boldsymbol{\alpha}(\mathbf{C})$ be used, but the closer it is to satisfying equation [subsurface_albedo_constraint] the closer it is to the ground truth. Ideally, this formula should take into account all the properties of the medium as well as the dielectric interface. A number of such approximate formulas have been derived in the literature. A well-known approximation is due to van de Hulst ([#Kulla2017], [#d'Eon2021]), which assumes an index-matched boundary (i.e. $\mathbf{E}_\mathrm{spec} = 0$, thus $\mathbf{E}_\mathrm{multi-scatter} = \mathbf{C}$), where:
 \begin{equation}
-\bar{\boldsymbol{\alpha}} = \frac{\boldsymbol{\alpha}} {1  - g (1 - \boldsymbol{\alpha})} \ ,
+\mathbf{C} = \frac{(1 - \mathbf{s})(1 - 0.139 \mathbf{s})}{1 + 1.17 \mathbf{s}} \ , \quad
+\mathbf{s} = \sqrt{\frac{1 - \boldsymbol{\alpha}}{1 - \boldsymbol{\alpha}\, g}} \ .
 \end{equation}
-and remapped extinction coefficient $\bar{\boldsymbol{\mu}}_t$ by
+Inversion of this gives the per-channel single-scatter albedo $\boldsymbol{\alpha}$ in terms of the **`subsurface_color`** $\mathbf{C}$ and anisotropy $g$:
 \begin{equation}
-\bar{\boldsymbol{\mu}}_t = \frac{1  - g (1 - \boldsymbol{\alpha})}{1 - g} \boldsymbol{\mu}_t \ .
+\boldsymbol{\alpha} = \frac{1 - \mathbf{s}^2}{1 - g\,\mathbf{s}^2} \ , \quad
+\mathbf{s}          = 4.09712 + 4.20863\mathbf{C} - \sqrt{9.59217 + 41.6808C + 17.7126\mathbf{C}^2} \ .
 \end{equation}
-Note that due to the singularity at $g=1$, in practice the anisotropy needs to be limited to be not too close to 1.
+This gives one approximate $\boldsymbol{\alpha}(\mathbf{C})$ mapping, but as noted it ignores the presence of the dielectric boundary. Other approximate forms which take into account the dielectric interface are presented in various sources (e.g. [#d'Eon2021], [#Burley2018]).
+We also note that the relevant approximation may depend on the specific assumptions made about the subsurface interface, for example in some cases renderers may approximate the boundary BSDF via a diffuse lobe, for efficiency and improved convergence. In that case, it would be more appropriate to use an $\boldsymbol{\alpha}(\mathbf{C})$ designed for this specific interface BSDF, in order to satisfy equation [subsurface_albedo_constraint].
+
+The phase function anisotropy $g$ of the medium is taken to be given directly by the **`subsurface_anisotropy`** parameter [^anisotropy_g]. It is assumed that the phase function has the standard Henyey-Greenstein form.
```

Comment:
> Henyey-Greenstein

Minor: Many style guides would recommend using en dashes instead of hyphens ("–" instead of "-") for relationships and connections like this one. However it seems fine to use hyphens as long as we're consistent about it.

There's more information about this here if anyone is interested: 
https://en.wikipedia.org/wiki/Dash#Relationships_and_connections

Reply:
In https://github.com/AcademySoftwareFoundation/OpenPBR/pull/171/commits/9311dfd99ee1133cf3f2791e5521c53754ffe425 I fixed up the hyphens to en dashes, for Henyey-Greenstein. We can make similar fixes elsewhere as needed.